brave/brave-core
# feat(wallet): implement bridge to aurora button

Resolves https://github.com/brave/brave-browser/issues/24377

## Submitter Checklist:

- [x] I confirm that no security/privacy review is not needed

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

1. Click on an asset in portfolio list for example Ethereum
2. On the asset details page, if the asset support aurora bridging, you will see a button with the text `Bridge to Aurora`
3. Click on the button, a modal should appear with bridging details.
4. Click on `Open rainbowbridge.app`. A new tab should open and load `https://rainbowbridge.app`

https://user-images.githubusercontent.com/48117473/182663673-444c9ecf-1dba-416d-88cc-23d7025b0bbd.mov
